### DISCUSSION TITLE
Actividad 0 - Sívori, Diego

### DISCUSSION THREAD

**Discussion by diego20366** [frlp-utn-ingsoft/catedra#65] [Show and tell]
https://github.com/diego20366/Actividad-0-Diego-Sivori.git

**Comment by RodrigoJacznik** [comment]
Hola,
Los commits podrian estar mejores (no sique el formato establecido).
La app no funciona actualizaste las `views` pero no las `urls`. Te falto esta parte https://docs.djangoproject.com/en/5.0/intro/tutorial04/#amend-urlconf
Por favor arregla la app para aprobar la actividad.

**Reply by diego20366** [reply]
Hola Rodrigo. Ya modifique las urls que me solicitaste.
Saludos

**Reply by RodrigoJacznik** [reply]
Pudiste probar la app antes de pushear? 
Creo que la db te quedo inconsistente, proba borrando el archivo db.sqlite3 y corre las migraciones (python manage.py migrate) y agrega una encuesta con opciones

**Reply by diego20366** [reply]
Profe ya hice lo del archivo db.sqlite3 y corri las migraciones. Tuve que hacer unas correcciones en el archivo models.py tanto en class questions y class choice porque me tiraba error cuando queria correr las migraciones. Pero lo pude solucionar.
Ahora consulta, hay algun comando en el que pueda correr y asi ver el usuario y  password que cree? porque me esta tirando error de contraseña o usuario y no se en que le estoy fallando. Necesito entrar para poder hacer lo de la encuesta.
Gracias. Saludos

**Reply by RodrigoJacznik** [reply]
Buenas, si borraste la db vas a tener que crear el superuser nuevamente

```python manage.py runserver```

**Reply by diego20366** [reply]
Hola profe, ahi ya hice las modificaciones pendientes.
Saludos

**Reply by ivanduva** [reply]
Buenas Diego.
Queda aprobado, pero fijate de mejorar los commits en las próximas actividades. La atomicidad está relativamente bien pero los mensajes no solo no siguen bien el formato sino que además tienen mensajes irrelevantes.
Por ejemplo este https://github.com/diego20366/Actividad-0-Diego-Sivori/commit/9871e3794120a75e9adce58a46d8f90d6c9cacbc habla mucho acerca del codigo, lo cual no es necesario porque mirando el diff del commit vemos que se modificó question_id por pk. No es necesario hablar demasiado del código en los mensajes.

Saludos
